src/app/public/modules/date-range-picker/types/date-range-calculator.ts

```diff
@@ -23,17 +23,33 @@ import {
 } from './date-range';
 
 export class SkyDateRangeCalculator {
+
+  /**
+   * Indicates the text displayed within the calculator select menu.
+   */
   public readonly shortDescription: string;
+
+  /**
+   * Indicates the calculator type.
```

Comment:
```suggestion
   * Specifies the type of calculations available for the date range.
```
How do `calculatorId` and `type` work together? Looking at the lists for `SkyDateRangeCaluclatorId` and `SkyDateRangeCalculatorType`, it seems like users would use one or the other, but I'm not sure. Do we need to explain how these two properties work together?

Reply:
`SkyDateRangeCalculatorType ` refers to the type of calculations that can be made. For example, a calculator with `calculatorId` of `After` uses a `SkyDateRangeCalculatorType.After`, while a `calculatorId` of `SpecificRange` uses `SkyDateRangeCalculatorType.Range`. I think the type is only important when you're creating your own custom calculator. It allows the consumer to define what type of calculation needs to be performed.